stable/jenkins/values.yaml

```diff
@@ -254,6 +254,8 @@ Master:
 
   ingress:
     enabled: false
+    # For Kubernetes v1.14+, use 'networking.k8s.io/v1'
+    ApiVersion: extensions/v1beta1
```

Comment:
please change to `apiVersion` ;)

Reply:
Tried to stay in Sync with the NetworkPolicy stuff... okay to derive from that?

Reply:
we are trying to go for a [best practice](https://helm.sh/docs/chart_best_practices/) chart

Reply:
Okie dokie!